src/new-client/src/App/LiveRates/Tile/Header/TileHeader.tsx

```diff
@@ -76,3 +83,25 @@ export const Header: React.FC = () => {
     </HeaderWrapper>
   )
 }
+
+export const HeaderComponent: React.FC<{
```

Comment:
The render of this is the same as `Header` above. 
To reduce duplication we should have a 'dumb' component that simply renders based on props, and then a 'container' component that passes props to the dumb component.

So for this example I would:
- Copy the original `Header` component, maybe call it `HeaderBase` or `HeaderInner` or `HeaderComponent`.
- Remove anything from the component that is a dependency and define those as the component's props. In this case it is data coming from the `useTileContext` hook, the `useDate` hook and the `tearOut` function
- Refactor the original `Header` component to use the new `HeaderInner` component, passing `currencyPair`, `supportsTearOut`, `date` and `onClick` as props.
- Now we could use the new `HeaderInner` component elsewhere outside the context of the application.

Reply:
[fixed here](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/1970/commits/bebfc79ca887b2478f776dfaba36f2ce1f214d5c)